README.md

```diff
@@ -26,6 +26,11 @@ to compile the test apps yourself.
 - **iOS**:
   - [Xcode](https://apps.apple.com/app/xcode/id497799835?mt=12) 11.3 or later
   - [CocoaPods](https://cocoapods.org/)
+- **Windows**:
+  - Ensure that Developer Mode is turned on in Windows Settings App
```

Comment:
We should also link to the documentation: https://docs.microsoft.com/en-us/windows/uwp/get-started/enable-your-device-for-development